<!DOCTYPE html>
<html>
<head>
  <title>Xenopus UCSC hub</title>
</head>

<body>

  <h1>Xenopus UCSC hub</h1>
  <h2>Description</h2>
  <p>The ChIP-sequencing tracks were produced as part of a project funded by NICHD to generate epigenome reference maps of Xenopus tropicalis embryos. Intergenic DNA 
    contains functional genomic elements which are largely unannotated. The biochemical features of chromatin, such as post-translational modifications of histone 
    proteins, determine its accessibility and function. The combination of different histone modifications can also be used to partition the genome in distinct 
    chromatin states, such as promoters, enhancers, transcribed regions, and repressed heterochromatin.</p>
  <h2>Methods</h2>
  <p>ChIP-sequencing was performed as described (Akkers et al. 2009, Akkers et al. 2012, Van Heeringen et al. 2014). The genome browser tracks have been generated 
    for the histone modifications H3K4me1, H3K4me3, H3K9ac, H3K9me2, H3K9me3, H3K27me3, H3K36me3 and H4K20me3, in addition to RNA polymerase II and p300. At five 
    stages of development  (NF stages 9, 10.5, 12.5, 16 and 30) they have been mapped with or without repetitive reads, to allow assessing histone modfications at 
    repetitive heterochromatic regions as well as unique regions within the genome. In addition they have been mapped to different X.tropicalis genome assemblies.</p>
  <h2>Using the track data hub</h2>
  <p>A description of the track hub can be found at the following location: <a href="http://veenstra.ncmls.nl/trackhub.htm">http://veenstra.ncmls.nl/trackhub.htm</a>
  Please contact Georgios Georgiou (email g.georgiou [@] science.ru.nl) for technical questions.</p>
  <h2>Credits and Data access policy</h2>
  <p>The data were generated in the Gert Jan Veenstra lab at Radboud University (The Netherlands). The track hub includes published data but also provides 
    pre-publication access to data. The unpublished data can be freely used on a per gene locus basis. Please contact Gert Jan Veenstra (email g.veenstra [@] science.ru.nl) 
    for all other use. </p>
  <h2>References</h2>
  <p>Akkers RC, Jacobi UG, Veenstra GJ. 2012. Chromatin immunoprecipitation analysis of Xenopus embryos. Methods Mol Biol. 2012;917:279-92. doi: 10.1007/978-1-61779-992-1_17. PMID: 22956095</p>
  <p>Akkers RC, van Heeringen SJ, Jacobi UG, Janssen-Megens EM, Fran√ßoijs KJ, Stunnenberg HG, Veenstra GJ. 2009. A hierarchy of H3K4me3 and H3K27me3 acquisition in spatial gene regulation in Xenopus embryos. Dev Cell. 2009 Sep;17(3):425-34. doi: 10.1016/j.devcel.2009.08.005. PMID: 19758566</p>
  <p>van Heeringen SJ, Akkers RC, van Kruijsbergen I, Arif MA, Hanssen LL, Sharifi N, Veenstra GJ. 2014. Principles of nucleation of H3K27 methylation during embryonic development. Genome Res. 2014 Mar;24(3):401-10. doi: 10.1101/gr.159608.113. Epub 2013 Dec 11. PMID: 24336765</p>
</html>
